### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by Swooty18**
изображение

### CODE REVIEW

**Review by marikrebega — APPROVED** [review]
Looks good)

**Review by IvanDanyliv — APPROVED** [review]

**Comment by chu4** [comment]
If i remember correctly, u need to install postgresql as well
that should be in readme

File: README.md [hunk 1]
```diff
@@ -1,27 +1,48 @@
 # README
+# OSBB
```
Comment (matsevich) [inline README.md:2]: можеш тут таке добавити

## Prerequisites

To run the project you need:

* Ruby `v.2.7.1`
* Node.js
* Yarn
* PostgreSQL

Comment (Swooty18) [reply README.md:2]: изображение
Додав

**Comment by Swooty18** [comment]
> If i remember correctly, u need to install postgresql as well
> that should be in readme

I am add prerequisites,it's ok?Or must add install postgresql?

**Review by matsevich — APPROVED** [review]

**Review by mmarusyk — COMMENTED** [review]
Add Contributing section.

**Comment by Swooty18** [comment]
> Add section about contributing.

Added

**Review by chu4 — APPROVED** [review]

**Pull request merged**